gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/controller/GraphControllerTest.java

```diff
@@ -95,7 +95,9 @@ void getGraphs_ReturnsGraphsAsList_whenSuccessful() throws Exception {
                 .url("my-graph-namespace.apps.k8s.cluster/ui")
                 .restUrl("my-graph-namespace.apps.k8s.cluster/rest")
                 .configName("mapStore")
-                .status(RestApiStatus.UP);
+                .status(RestApiStatus.UP)
+                .graphAutoDestroyDate("2022-06-09t15:55:34.006");
```

Comment:
This is in the past so will it destroy straight away? Also it seems strange to pick an arbitrary date

Reply:
The graph is auto destroyed when is auto destroy date is equal current time or older

Reply:
Maybe this can be current date or something then? Rather than an arbitrary date?